types/node/http2.d.ts

```diff
@@ -651,7 +651,7 @@ declare module "http2" {
     }
 
     export class Http2ServerResponse extends stream.Stream {
-        private constructor();
+        constructor();
```

Comment:
The arguments are missing, see https://github.com/nodejs/node/blob/0481a7f3b883862cadf963314d23490adc6656a4/lib/internal/http2/compat.js#L428

Reply:
I added type for the first argument but second "options" I am not sure what should be used. It seems options are passed to superclass Stream. Stream class do not define specific constructor options ?

Reply:
it is bug in node? this constructor should be empty ?

Reply:
why empty? see https://github.com/nodejs/node/blob/0481a7f3b883862cadf963314d23490adc6656a4/lib/internal/http2/compat.js#L743 the first argument is a `ServerHttp2Stream`. I think the second arg can be omitted.

Reply:
It is already like that. Thanks for confirming the second arg.